typescript/sdk/src/metadata/chainMetadataTypes.ts

```diff
@@ -245,6 +245,7 @@ export type BlockExplorer = Exclude<
 >[number];
 
 export function isValidChainMetadata(c: ChainMetadata): boolean {
+  ChainMetadataSchema.parse(c);
```

Comment:
`is*` functions like this shouldn't throw IMO, or the return value is never false